server/modules/analysis/service/rChainService/rCalculation.js

```diff
@@ -47,7 +50,10 @@ export default class RCalculation {
         NodeDef.getName
       )(this.calculation)
 
-      await this.rFile.appendContent(setVar(dfVar(entityName, attributeName), NA))
+      const calculationUuid = ProcessingStepCalculation.getUuid(this.calculation)
+      const script = await CalculationManager.fetchCalculationScript({ surveyId, calculationUuid })
```

Comment:
CalculationManager should not exist, neither the method CalculationManager.fetchCalculationScript.
In rChain init, use fetchChain with includeStript = true and you already have the script associated to the calculation

Reply:
ok, great